src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,35 @@
+.stars {
+  height: 16px;
+  display: flex;
+  flex-direction: row;
+
+  &__star-active {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star-active.svg);
+    box-sizing: border-box;
+    align-items: center;
+    margin: 0 4px 0 0;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    box-sizing: border-box;
+    align-items: center;
+    margin: 0 4px 0 0;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &__star-active:last-child {
+    margin-right: 0;
+  }
+
+  &__star:last-child {
+    margin-right: 0;
```

Comment:
The ':last-child' pseudo-class can be combined to reduce redundancy. You can write a single rule '.stars__star:last-child, .stars__star-active:last-child' to set the 'margin-right' to 0 for the last child of both types.